frontend/src/__tests__/cypress/cypress/pages/servingRuntimes.ts

```diff
@@ -123,6 +123,22 @@ class ServingRuntimes {
   getRowById(id: string) {
     return new ServingRuntimeRow(id);
   }
+
+  findEditModel() {
+    return cy.findByLabelText('Kebab toggle');
```

Comment:
The "Kebab togle" returns the Edit functionality?

Reply:
Yes, that's what the element is called at present.